docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md

```diff
@@ -134,3 +134,96 @@ console data:import glossary
 ## Retrieving and using payment details from Stripe
 
 For instructions on using payment details, like the payment reference, from Stripe, see [Retrieve and use payment details from third-party PSPs](https://docs.spryker.com/docs/pbc/all/payment-service-provider/{{page.version}}/base-shop/retrieve-and-use-payment-details-from-third-party-psps.html)
+
+## Using Stripe in Marketplace context
+
+Install the required modules:
+
+```bash
+composer require spryker/merchant-app spryker/merchant-app-merchant-portal-gui
+```
+
+### Configuration
+
+To enable your application to work with Stripe in a Marketplace context, you need to configure your application by adding some plugins, update some configurations, and update your state-machine configuration.
+
+#### Plugins
+
+##### KernelApp
+- `\Spryker\Zed\MerchantApp\Communication\Plugin\KernelApp\MerchantAppRequestExpanderPlugin` - Add this one to `\Pyz\Zed\KernelApp\KernelAppDependencyProvider::getRequestExpanderPlugins()`
+- `\Spryker\Zed\OauthClient\Communication\Plugin\KernelApp\OAuthRequestExpanderPlugin` - Add this one to `\Pyz\Zed\KernelApp\KernelAppDependencyProvider::getRequestExpanderPlugins()`
```

Comment:
config_default.php changes are missing required for these plugins:
-  $config[OauthClientConstants::OAUTH_PROVIDER_NAME_FOR_ACP]
- $config[OauthClientConstants::OAUTH_GRANT_TYPE_FOR_ACP] 
- $config[OauthClientConstants::OAUTH_OPTION_AUDIENCE_FOR_ACP]
- $config[KernelAppConstants::TENANT_IDENTIFIER]